static/src/javascripts/projects/common/views/content/static-facebook.html

```diff
@@ -0,0 +1,10 @@
+<div class="social-fixed__container" data-component="static-social">
```

Comment:
__ is the "child" syntax, so `social-fixed__container ` would imply it is a child of the `social-fixed` element. Check out some explanations of BEM syntax e.g. http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/